packages/editor/src/components/post-card-panel/index.js

```diff
@@ -64,7 +64,7 @@ export default function PostCardPanel( {
 		[ postId, postType ]
 	);
 
-	const pageTypeBadge = usePageTypeBadge();
+	const pageTypeBadge = usePageTypeBadge( parseInt( postId, 10 ) );
```

Comment:
Why do we need to parse the postId to an integer? Isn't it already one? Or perhaps the previous `getCurrentPostId` returned an integer and the current method (through props somewhere) returns a string?

Reply:
Not in the PostEditHeader component where we render this, but it is an integer in the `PostSummary`.
I will remove the parsing here and move that up one level to the header. As we know its a string there for sure.

Reply:
Just noting that postId is something a real string (templates and template parts). It's true that quick edit is only used for "posts" and "pages" now but `PostCardPanel` can be used for templates and template parts too.

Reply:
Yeah I moved it up to the `PostEditHeader`: https://github.com/WordPress/gutenberg/pull/67390/commits/5e2d674c986aaf2109ed455f38927d17124641ba

The `usePageTypeBadge` hook works fine whether passing an integer or string, but does rely on integer for the home/posts page match.